### ISSUE TITLE
Device limit not being recognized

### ISSUE THREAD

**Issue by digitalcrisis** [fusionpbx/fusionpbx#2816]
When you limit the number devices permitted on a per domain basis, it is not being enforced when you add a device via the edit extensions screen.

**Comment by markjcrane** [comment]
Did you clear the session? And did you login with a user account on that domain?

**Comment by digitalcrisis** [comment]
Yes, I've used a different browser, flushed cache on the browsers to double check, please understand I'm not trying to add the devices via the devices page, but the extensions page, where you can edit the extension and add the device there....
